### ISSUE TITLE
RuntimeError: No such middleware to insert before: Rails::Rack::Logger

### ISSUE THREAD

**Issue by BaneOfSerenity** [stve/silencer#21]
Running Rails 5.0.0.1 with silencer 1.0.0.rc3 and having the same issues as in issue #9

**Comment by stve** [comment]
Hi @BaneOfSerenity,

Quite a few have had issues with this and I hope to get to the bottom of this soon. In the meantime, a workaround has been identified which is to move the configuration to an initializer (ie, config/initializers/silencer.rb):

```
Rails.application.configure do
  config.middleware.swap Rails::Rack::Logger, Silencer::Logger, :silence => ["/noisy/action.json"]
end
```

Can you give that a try? Apologies for the trouble, I hope to have this figured out shortly. Thanks!

**Comment by stve** [comment]
@BaneOfSerenity I've just released `1.0.0`, feel free to re-open if you run into further issues. Thanks!
